src/libutil/config.cc

```diff
@@ -123,7 +122,10 @@ void AbstractConfig::applyConfig(const std::string & contents, const std::string
                 throw UsageError("illegal configuration line '%1%' in '%2%'", line, path);
             auto p = absPath(tokens[1], dirOf(path));
             if (pathExists(p)) {
-                applyConfigFile(p);
+                try {
+                    std::string includedContents = readFile(path);
+                    applyConfigInner(includedContents, p, parsedContents);
+                } catch (SysError &) { }
```

Comment:
I added the try/catch so it matches `applyConfigFile`, not sure it's needed

Reply:
Yeah good call. It does technically make a difference if the file has funny perms.

Write a TODO for "do we actually want this? or is it better to fail?" maybe? I think it was an accident. But best actually removed in a separate PR.